cycamore/build.sh

```diff
@@ -1,18 +1,30 @@
 #!/bin/bash
 
-mkdir build; cd build;
+mkdir build; 
+cd build;
 export LD_LIBRARY_PATH=$PREFIX/lib/
 export CMAKE_LIBRARY_PATH=$PREFIX/lib/
 export PATH=$PREFIX/bin:$PATH
 export MACOSX_DEPLOYMENT_TARGET=
+
+# # hack fix for hdf5 issues
+# if [[  `uname` == 'Linux' ]]; then
+#   ln -s $PREFIX/lib/libhdf5.so.9 $PREFIX/lib/libhdf5.so.8
+#   ln -s $PREFIX/lib/libhdf5_hl.so.9 $PREFIX/lib/libhdf5_hl.so.8
+# else
+#   ln -s $PREFIX/lib/libhdf5.9.dylib $PREFIX/lib/libhdf5.8.dylib
+#   ln -s $PREFIX/lib/libhdf5_hl.9.dylib $PREFIX/lib/libhdf5_hl.8.dylib
+# fi
+
 $PREFIX/bin/cyclus --version
+
 if [[  `uname` == 'Linux' ]]; then
 
-    cmake ..  -DCMAKE_INSTALL_PREFIX=$PREFIX -DCYCLUS_ROOT_DIR=$PREFIX -DBOOST_ROOT=$PREFIX  -DBOOST_LIBRARYDIR=$PREFIX/lib -DBoost_NO_SYSTEM_PATHS=ON -DCMAKE_BUILD_TYPE=Release -DLAPACK_LIBRARIES=$PREFIX/lib/liblapack.so -DBLAS_LIBRARIES=$PREFIX/lib/libblas.so
+    cmake ..  -DCMAKE_INSTALL_PREFIX=$PREFIX -DCYCLUS_ROOT_DIR=$PREFIX -DHDF5_ROOT=$PREFIX -DBOOST_ROOT=$PREFIX  -DBOOST_LIBRARYDIR=$PREFIX/lib -DBoost_NO_SYSTEM_PATHS=ON -DCMAKE_BUILD_TYPE=Release -DLAPACK_LIBRARIES=$PREFIX/lib/liblapack.so -DBLAS_LIBRARIES=$PREFIX/lib/libblas.so
```

Comment:
I explicitly left comments for whomever comes next with the hdf5 issue. If
you really want them gone, I can remove them, but I think the possible
future help more than makes up for any detraction.

On Thu, Apr 23, 2015 at 12:28 PM, Anthony Scopatz <notifications@github.com>
wrote:

> In cycamore/build.sh
> <https://github.com/cyclus/ciclus/pull/62#discussion_r28983940>:
>
> >  if [[  `uname` == 'Linux' ]]; then
> >
> > -    cmake ..  -DCMAKE_INSTALL_PREFIX=$PREFIX -DCYCLUS_ROOT_DIR=$PREFIX -DBOOST_ROOT=$PREFIX  -DBOOST_LIBRARYDIR=$PREFIX/lib -DBoost_NO_SYSTEM_PATHS=ON -DCMAKE_BUILD_TYPE=Release -DLAPACK_LIBRARIES=$PREFIX/lib/liblapack.so -DBLAS_LIBRARIES=$PREFIX/lib/libblas.so
> > +    cmake ..  -DCMAKE_INSTALL_PREFIX=$PREFIX -DCYCLUS_ROOT_DIR=$PREFIX -DHDF5_ROOT=$PREFIX -DBOOST_ROOT=$PREFIX  -DBOOST_LIBRARYDIR=$PREFIX/lib -DBoost_NO_SYSTEM_PATHS=ON -DCMAKE_BUILD_TYPE=Release -DLAPACK_LIBRARIES=$PREFIX/lib/liblapack.so -DBLAS_LIBRARIES=$PREFIX/lib/libblas.so
>
> line breaks would be nice :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/ciclus/pull/62/files#r28983940>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192